config/app.ts

```diff
@@ -2,9 +2,18 @@
  * Application related configuration
  */
 export default {
-	/** 
+  /**
 	The port the web server listens on for connections.
 	*/
-	port: process.env.PORT ? process.env.PORT : 3000,
-	ldap: process.env.LDAP ? process.env.LDAP : "IDK"
-}
\ No newline at end of file
+  port: process.env.PORT ? process.env.PORT : 3000,
+  ldap: process.env.LDAP ? process.env.LDAP : "IDK",
+  ticket_id: process.env.TICKET_ID ? process.env.TICKET_ID : "wrong",
+  ticket_uuid: process.env.TICKET_UUID ? process.env.TICKET_UUID : "hahaha",
+  url: process.env.URL ? process.env.URL : "move along",
+  nodename: process.env.NODENAME ? process.env.NODENAME : "don't mind me",
```

Comment:
I'm not sure what "IDK", "move along", "don't mind me" etc do in this block but I'm sure it's ok lol.